### ISSUE TITLE
erysipelas-like erythema

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#11640]
**Preferred term label:**
erysipelas-like erythema
-- description of the cutaneous lesions seen in familial mediterranean fever.

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6001541
- label: Erysipelas-like erythema
- parents: [Erythema (HP:0010783)](https://hpo.jax.org/app/browse/term/HP:0010783)
- pmid: PMID:22243424
- def: A red, warm and tender rash, usually involving the distal part of the shin, dorsum of the foot or ankle, which typically fades, within 24 to 72 hours. A typical lesion of erysipeloid erythema is comprised of an erythematous plaque with well-defined borders located on the lower extremity. Histologically one may expect a perivascular and interstitial inflammatory infiltrate containing neutrophils and nuclear dust.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11640
- synonyms: Pseudoerysipelas(NONE)|Erysipeloid erythema(NONE)
